### DISCUSSION TITLE
Welcome to refinery Discussions!

### DISCUSSION THREAD

**Discussion by jhoetter** [code-kern-ai/refinery#25] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jhoetter** [comment]
Starting with myself, I'm Johannes, one of the co-founders of [Kern AI](https://www.kern.ai). I'm a data engineer and scientist by heart, and I love talking about software and developer tools for hours. My first AI touch point was [style transfer](https://www.tensorflow.org/tutorials/generative/style_transfer) roughly six years ago, and I fell in love with the techniques behind it immediately. Before founding Kern AI, I worked as an AI consultant together with my dear co-founder @WeHenrik.

Excited to hear more about the people from the community! Cheers ✌️

**Comment by JWittmeyer** [comment]
Hello there,

general K… oh damn, kind of a reflex by this point in time.
Hi :smiley: . I’m Jens, part of the kern.ai team since last year and responsible for quite a few lines of code. I’m passionate about programming, acting, and pretty much everything in between. You might even find a minigame or two I build myself.

**Comment by mishanti** [comment]
Hello everyone,

nice to see you here! I'm Mischa and I take care of kern's cloud infrastructure. I am always eager to discuss kern deployment issues on various platforms, MLOps and GPU builds :racing_car::dash:

**Comment by DerEchteFeuerpfeil** [comment]
# New challenger approaching - Moritz

Hello Community!
My Name is Moritz and I am a working student @ kern.ai doing developer relations. I will hang out in these discussions a lot, and am looking forward to hearing from you about your experiences, difficulties, and success stories :blush:

# Background & Interests

I am studying Data Engineering in my Master’s right now and kind of specialize in Machine Learning, NLP, and data-centric AI methods.
I am also really interested in AI-Art, exploring large language models, and hacking together small ML Apps. So if you’re interested in any of those, feel free to reach out to me, maybe we can share experiences or even just opinions :wink:

# Beyond the computer

I’ll keep it short: Biking, Beach Volleyball, and hanging out. In future meetups I’ll make sure to bring a volleyball, so you can challenge the kern team :volleyball:

**Comment by LeonardPuettmann** [comment]
Hello guys :wave:
my name is Leo and I am a Data Scientist & working student at Kern! For me, the wonder of Data Science and AI is that it allows you to see the world from a different perspective. The first touchpoint I had with AI was in an amazing class at my university. Scince then I knew that I wanted to work with data and AI! I am super happy to be able to support Kern in their vision to make the world of data easier, faster and more fun to use.

In my spare time I like to hit the gym or enjoy good food with friends. And I also have a deep, affectionate love for high quality tea. :teapot:

**Comment by lumburovskalina** [comment]
Hello people!

My name is Lina and I am a Front-end Developer at Kern. With my experience and skills in Angular, Typescript and Tailwind I love turning even the craziest mockup into reality and placing every element perfectly. At Kern I work on the software’s design, general front-end issues and constantly try to improve the code quality.

In my time off work, I like to clear my mind and connect with the nature on long hikes. Additionally, I love taking part in running events over various distances. Other than that, I like to broaden my skill set by learning Python and deepen my understanding of programming.

I am so happy that I joined Kern and I am looking forward to all the challenges!

**Comment by SirDegraf** [comment]
Hi there ✌️,
my name is Simon and I am a dev here at kern. In my bachelor’s studies in the field of business information systems, I first got to know about the importance of high-quality data for the future of successful companies. Now we are giving our best to enable our community to make great use of their data and I love to be a part of this journey. I also like to lift a lot of differently shaped weights 🏋️‍♂️ and eat a lot of delicious foods 🥙. For questions about the journey we are creating, the weights I lift or the food I eat reach out to me!

**Comment by FelixKirsch** [comment]
Hello everybody 👋
I’m Felix, one of the developers at Kern. My topics are mainly AI related, but I think I touched at least a line of code in most of the services 🧐 I’m happy to help if you experience any problems or come up with any question!

Besides coding, I like to spend time with friends, while doing sports together or visiting the next bar or cafe ☕
